coffee_lib/src/plugin.rs

```diff
@@ -94,7 +111,7 @@ impl Plugin {
     /// configure the plugin in order to work with cln.
     ///
     /// In case of success return the path of the executable.
-    pub async fn configure(&mut self) -> Result<String, CoffeeError> {
+    pub async fn configure(&mut self, verbose: &bool) -> Result<String, CoffeeError> {
```

Comment:
```suggestion
    pub async fn configure(&mut self, verbose: bool) -> Result<String, CoffeeError> {
```

Yo do not need to borrow a bool :) it is fast to copy it